tests/lib/mozApps.js

```diff
@@ -43,14 +59,20 @@ function initialize() {
                 var request = {};
 
                 setTimeout(function() {
-                    if (request.onsuccess && request.onsuccess.constructor === Function) {
+                    if (window._.isFunction(request.onsuccess)) {
+                        console.log('[mozApps] Calling request.onsuccess');
                         request.onsuccess();
+                    } else {
+                        console.log('[mozApps] Not calling request.onsuccess');
+                        console.log(request.onsuccess);
                     }
                 });
 
                 return request;
             },
             install: function(manifest) {
+                console.log('[mozApps] Installing app');
+                console.log(manifest);
```

Comment:
i feel most of these logs will cause a lot of noise. are the install tests finnicky?